src/js/components/Select/stories/typescript/CreateOption.tsx

```diff
@@ -1,19 +1,20 @@
 import React, { useState } from 'react';
 import { Box, Select } from 'grommet';
-
+import { OptionType } from '../../index';
 // the prefix name of the Create option entry
 const prefix = 'Create';
 
-const defaultOptions = [];
+const defaultOptions: OptionType[] = [];
 for (let i = 1; i <= 5; i += 1) {
   defaultOptions.push(`option ${i}`);
 }
 
 const updateCreateOption = (text: string) => {
   const len = defaultOptions.length;
-  if (defaultOptions[len - 1].includes(prefix)) {
-    // remove Create option before adding an updated one
-    defaultOptions.pop();
+  if (typeof defaultOptions[len - 1] === 'string') {
```

Comment:
Can we simplify logic to `if (typeof defaultOptions[len - 1] === 'string' && defaultOptions[len - 1].includes(prefix))`?